frontend/__mocks__/k8sResourcesMocks.ts

```diff
@@ -74,12 +76,12 @@ export const testClusterServiceVersion: ClusterServiceVersionKind = {
           kind: 'Pod',
           version: 'v1',
         }],
-        specDescriptors: [{
-          path: 'size',
-          displayName: 'Size',
-          description: 'The desired number of Pods for the cluster',
-          'x-descriptors': [SpecCapability.podCount],
-        }],
+        specDescriptors: Object.keys(SpecCapability).filter(c => !prefixedCapabilities.has(SpecCapability[c])).map(capability => ({
```

Comment:
Add implemented `specDescriptor` capabilities to the mock `ClusterServiceVersion` object for testing.